### DISCUSSION TITLE
Installation / Configuration Datenbank (database)

### DISCUSSION THREAD

**Discussion by AndyPre7** [OpenDTU-App/opendtu-react-native#23] [Q&A] [answered]
Tolle App!

Frage:
Wo genau lege ich denn die Datenbank hin, er fragt nach einer URL (2. Feld beim Anlegen der Datenbank)?

Derzeit sehe ich nur den aktuellen Ertrag sowie die Leistung. Darunter eben die Meldung der fehlenden Datenbank.

Bin totaler Anfänger, bitte um ein wenig Rücksicht, falls ich offensichtliches übersehen habe.

**Comment by CommanderRedYT** [comment, marked as answer]
Da musst du die URL zur Prometheus Datenbank angeben. OpenDTU hat da nativen Support für

**Reply by AndyPre7** [reply]
🤔😇
Wo finde ich den Pfad dorthin heraus?
Liegt diese auf meiner OpenDTU?
Habe die Version Open DTU onBatterie falls das hilft.
Sorry, gibt es einen Weg, sich zügig einzulesen?
Danke!

**Reply by CommanderRedYT** [reply]
Das kommt ganz darauf an wo du Prometheus installiert hast (beispielsweise Raspberry Pi)

**Reply by AndyPre7** [reply]
Ok, da bin ich weit von entfernt. Einen raspi habe ich noch gar nicht.
Es kommt Licht ins Dunkle, vielen Dank!
